packages/react-dom/src/__tests__/ReactTestUtilsAct-test.js

```diff
@@ -779,13 +779,10 @@ function runActTests(render, unmount, rerender) {
         });
       }
     });
-    describe('throw in prod mode', () => {
-      // @gate !__DEV__
-      it('warns if you try to use act() in prod mode', () => {
-        expect(() => act(() => {})).toThrow(
-          'act(...) is not supported in production builds of React',
-        );
-      });
+
+    // @gate !__DEV__
+    it('is not available in prod mode', () => {
+      expect(() => act(() => {})).toThrow('act is not a function');
```

Comment:
`react-dom/test-utils` is deprecated so I didn't bother with a separate entrypoint.